create_vm.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+usage() {
+  cat <<EOF
+  usage: $0 [-r]
+  This script creates
+    1) an empty VM with CentOS 6.8 ready to be installed, or
+    2) CentOS 6.8 VM when -r parameter is applied (reuses vdi from vdis/).
+  OPTIONS:
+    -h Show this message
+    -r Reuses a vdi that already exists.
+EOF
+}
+
+create_vm() {
+  echo -e "Creating an empty vm..."
+  source scripts/create_empty_vm.sh
+
+  if [[ -z "$reuse_vdi" ]];then
+    echo -e "\nCreating an empty vdi..."
+    source scripts/create_empty_vdi.sh
+  else
+    echo -e "\nCopying an existing vdi..."
+    VM="Centos-68-64bit"
+    cp vdis/$VM.vdi .
```

Comment:
VM here is always "Centos-68-64bit" and it is not used in any other place, so I would just use the VM name and remove the VM variable.
I usually prefer full paths instead of relative ones, so I avoid ./ or ../ or ~/ and instead always indicate the full path where the things need to run, be copied to, etc...